Sprig App - Star Snatcher

sudo-apt-install-tap opened the pull request:
**Author: TAP**

**Collect Each Stars On Every Level**

**Use WASD and play :)**

sudo-apt-install-tap committed:
Sprig App - Star Snatcher

sudo-apt-install-tap committed:
Merge branch 'main' into Automated-PR-1765638586571

sudo-apt-install-tap committed:
Revise game metadata and clean up code

Updated metadata and removed redundant setMap call.

theamazing0 commented:
<img width="351" height="76" alt="image" src="https://github.com/user-attachments/assets/d4a3c963-4d1c-4fca-b623-09ae9f4dc033" />

Error seems to persist, please make sure to test your code before submitting

sudo-apt-install-tap committed:
Update game metadata and level map

Removed tags from the game metadata and made a minor change to the level map.

sudo-apt-install-tap committed:
Update wall and star bitmap patterns

sudo-apt-install-tap committed:
Merge branch 'main' into Automated-PR-1765638586571

sudo-apt-install-tap reviewed (commented):
Can you check this? Thanks!!

sudo-apt-install-tap committed:
Merge branch 'main' into Automated-PR-1765638586571

whatwareweb commented:
Hi! It doesn't look like the changes are showing, you can check your files and edit them from here:
https://github.com/hackclub/sprig/pull/3539/changes

sudo-apt-install-tap committed:
Fix formatting of description in Star-Snatcher.js

sudo-apt-install-tap committed:
Merge branch 'main' into Automated-PR-1765638586571

whatwareweb commented:
Hello! Game looks good and i love the art! Can you add some more levels and game mechanics?
Specifically, this game does not meet the 1 minute time requirement, and also the unique gameplay.

sudo-apt-install-tap commented:
Will do!

sudo-apt-install-tap committed:
Merge branch 'main' into Automated-PR-1765638586571

whatwareweb commented:
If you made any changes, it doesn't seem like they made it to the files tab - https://github.com/hackclub/sprig/pull/3539/changes

sudo-apt-install-tap committed:
Enhance Star-Snatcher game with new features

Updated game description and added new elements like gems and moons. Implemented lives system and modified level progression.

sudo-apt-install-tap committed:
Merge branch 'main' into Automated-PR-1765638586571

sudo-apt-install-tap commented:
heya! changed a bunch of things! Added some mechanics. P.S avoid the lava it resets you back to the start of the level

SSoggyTacoMan reviewed (approved):
Hey there! Really cool game, Great job!

Now you wait for a maintainer to check the game out and approve your game :)

whatwareweb commented:
Thanks for your submission!

If you're a teen, you can get your Sprig here! [sprig-order.hackclub.dev](https://sprig-order.hackclub.dev/)

Hack Clubbers would love to see this! Just post a link and 1-2 sentences talking about your game to our [#ship](https://hackclub.slack.com/archives/C0M8PUPU6) [channel](https://hackclub.com/slack/)
